pkg/kubelet/dockershim/docker_service.go

```
@@ -197,6 +197,9 @@ func NewDockerService(client dockertools.DockerInterface, seccompProfileRoot str
 	} else {
 		cgroupDriver = dockerInfo.CgroupDriver
 	}
+	if dockerInfo.LoggingDriver != "json-file" {
```

warn if its not detectable, fail hard if it is knowable.